addons/web/static/src/public/colibri.js

```diff
@@ -15,6 +15,7 @@ export class Colibri {
         this.isReady = false;
         this.isUpdating = false;
         this.isDestroyed = false;
+        this.INITIAL_VALUE = Symbol();
```

Comment:
I got this suggestion from Géry: making the constant `static`.
```js
class Interaction {
  static INITIAL_VALUE = new Symbol("initial_value");
}
```
Maybe define it as static in both classes (reusing the value).